### PR TITLE
TWave: various documentation fixes

### PR DESCRIPTION
**Pull request by waldyrious**
- fix math rendering (was missing a space after backtick)
- include units in the documentation of properties, for additional clarity
- use articles ("a", "the"...) consistently
- minor punctuation tweaks for correctness and readability

### CODE REVIEW

File: sympy/physics/optics/waves.py [hunk 1]
```diff
@@ -193,8 +198,9 @@ def phase(self):
     @property
     def speed(self):
         """
-        Returns the speed of travelling wave.
-        It is medium dependent.
+        Returns the propagation speed of the wave,
+        in meters mer second.
```
Comment (smichr) [inline sympy/physics/optics/waves.py]: mer -> per

Comment (waldyrious) [inline sympy/physics/optics/waves.py]: oops :) will fix.

**Comment by smichr** [comment]
+1

**Comment by smichr** [comment]
Thanks -- it's in.

**Pull request merged**